worker/jobTypes/githubJob.js

```diff
@@ -213,8 +230,29 @@ class GitHubJobClass {
         // check if need to build next-gen
         if (this.buildNextGen()) {
             commandsToBuild[commandsToBuild.length - 1] = 'make next-gen-html';
+           }
+         //overwrite gatsby config file
+        if (currentJob.payload.jobType === 'productionDeploy'){
```

Comment:
This conditional needs a check for next gen as well, perhaps wrap in the previous.